.github/workflows/go.yml

```diff
@@ -0,0 +1,25 @@
+name: Go
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: "1.20"
```

Comment:
How about using a matrix strategy like we do here https://github.com/luno/reflex/blob/master/.github/workflows/go.yml ?